unity-client/Assets/Scripts/MainScripts/DCL/DataStorage/DataStorage.cs

```diff
@@ -0,0 +1,7 @@
+﻿using Variables.RealmsInfo;
+
+public static class DataStorage
```

Comment:
```suggestion
public static class DataStore
```

Maybe `DataStore` is a better option, as kernel uses the same word for its store.